tests/containers/podman_pods.pm

```diff
@@ -78,7 +78,7 @@ sub run {
         validate_script_output('podman pod ps', sub { !m/testing-pod/ });
         validate_script_output('podman ps', sub { !m/testing-pod-container/ });
 
-        if (check_min_runtime_version('4.4.0')) {
+        if (check_min_runtime_version('4.4.0') && !is_staging) {
```

Comment:
Ok, but if this is a temporary fix, please add a comment that reminds us to work on a proper solution.

Reply:
I don't think it's temporary. If we can't install certain packages cause in staging we don't have full repos, we can live without this test in staging.

Reply:
Agree with Jose.